src/main/java/frc/robot/Robot.java

```diff
@@ -64,12 +66,13 @@ public void simulationPeriodic() {}
 
   // Controller Configuration
```

Comment:
Calling a method “configure…” implies that it is called only once, or infrequently. This method is called every periodic loop. Consider renaming it… something like “periodicIntakeControls” would be more appropriate.

Reply:
That's logical, I'll update the naming.